stacks-signer/src/signer.rs

```diff
@@ -1605,6 +1656,37 @@ impl Signer {
     }
 }
 
+/// Convert the signer state to a string
+pub fn signer_state_to_string(state: &SignerState) -> String {
```

Comment:
Nit: can these methods here instead be part of a `Display` impl for `SignerState` and `CoordinatorState`?